config/initializers/config.rb

```diff
@@ -0,0 +1,58 @@
+Config.setup do |config|
+  # Name of the constant exposing loaded settings
+  config.const_name = 'Settings'
```

Comment:
```suggestion
  config.const_name = "Settings"
```
check lại toàn bộ nha em

Reply:
Dạ oke a